src/components/search/search.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { SearchBar } from "@nypl/design-system-react-components";
+import { useRouter } from "next/router";
+import PublicDomainFilter from "../publicDomainFilter/publicDomainFilter";
+
+const Search = () => {
+  const router = useRouter();
+  const [keywords, setKeywords] = useState("");
+  const [publicDomainOnly, setPublicDomainOnly] = useState(false);
+
+  const handleSubmit = (event, router) => {
+    event.preventDefault();
+    const searchUrl =
```

Comment:
Prod DC adds `utf8=✓` to the query string in the URL (https://digitalcollections.nypl.org/search/index?utf8=%E2%9C%93&keywords=cat&filters%5Brights%5D=pd). It's the only thing missing in this string.